src/components/bookDetails/BookDetails.css

```diff
@@ -0,0 +1,104 @@
+
+.container_for_details {
+    width: 375px;
+    margin-top: 25px;
+    height: 800px;
+}
+
+.container_for_img {
+    width: 30px;
+    height: 30pxs;
+}
+
+.card_img_for_book_details {
+    margin-left: 65px;
+    width: 250px;
+    height: 250px;
+}
+
+.button_back {
+    position: absolute;
+    right: 10px;
+    top: 10px;
+    border-radius: 5px;
+    background: #f6f62c;
+}
+@media only screen and (min-width: 750px  ) {
+
+.container_for_details {
+    width: auto;
+    height: auto;
+    gap: 20px;
+    background: #d6f3f8;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.container_for_button {
+    display: flex;
+    justify-content: space-around;
+    margin-top: 100px;
+}
+
+.container_for_img {
+    
+}
+.card_img_for_book_details {
+    display: flex;
+    margin-top: 100px;
+    margin-left: 100px;
+}
+
+.button_edit {
+    background: #6969e4;
+    height: 40px;
+    width: 150px;
+    border-radius: 5px;
+}
+
+.button_delete {
+    background: #ff3333;
+    height: 40px;
+    width: 150px;
+    border-radius: 5px;
+}
+
+.button_rent {
+    background: #bef0be;
+    height: 40px;
+    width: 150px;
+    border-radius: 5px;
+}
+
+.container_for_text {
+    display: block;
+}
+
+.card_img_for_book_details {
+    border: inset 3px #6b6bcb;
+    height: 500px;
+    width: 500px;
+}
+
+.container_for_quantity {
+    background-color: rgb(151, 221, 46);
+    width: 130px;
+}
+
+.button_back {
+    border-radius: 5px;
+    position: absolute;
+    right: 50px;
+    width: 40px;
+    height: 30px;
+    top: 50px;
+    background: yellow;
+}
+
+.container_for_button {
+    display: flex;
+    position: absolute;
+    bottom: 100px;
+    justify-content: space-between;
+}
+}
```

Comment:
?

Reply:
All css file for BookDetails component is wrong. I must fix this.